src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -2117,17 +2116,37 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_BadNewExpr, "this").WithLocation(8, 21),
                 // (8,21): error CS1002: ; expected
                 //             e = new this;   // CS1031, not a type
-                Diagnostic(ErrorCode.ERR_SemicolonExpected, "this").WithLocation(8, 21),
-                // (9,21): error CS8181: 'new' cannot be used with tuple type. Use a tuple literal expression instead.
-                //             e = new ();     // CS1031, too few tuple elements
-                Diagnostic(ErrorCode.ERR_NewWithTupleTypeSyntax, "()").WithLocation(9, 21),
-                // (9,22): error CS8124: Tuple must contain at least two elements.
-                //             e = new ();     // CS1031, too few tuple elements
-                Diagnostic(ErrorCode.ERR_TupleTooFewElements, ")").WithLocation(9, 22),
-                // (9,23): error CS1526: A new expression requires (), [], or {} after type
-                //             e = new ();     // CS1031, too few tuple elements
-                Diagnostic(ErrorCode.ERR_BadNewExpr, ";").WithLocation(9, 23)
-             );
+                Diagnostic(ErrorCode.ERR_SemicolonExpected, "this").WithLocation(8, 21));
+        }
+
+        [Fact]
+        public void CS1031ERR_TypeExpected02_Tuple()
+        {
+            var text = @"namespace x
+{
+    public class a
+    {
+        public static void Main()
+        {
+            var e = new ();
+        }
+    }
+}
+";
+            // TODO: this appears to be a severe regression from Dev10, which neatly reported 3 errors.
+            CreateCompilationWithMscorlib(text).VerifyDiagnostics(
+                // (7,26): error CS8124: Tuple must contain at least two elements.
+                //             var e = new ();
+                Diagnostic(ErrorCode.ERR_TupleTooFewElements, ")").WithLocation(7, 26),
+                // (7,27): error CS1526: A new expression requires (), [], or {} after type
+                //             var e = new ();
+                Diagnostic(ErrorCode.ERR_BadNewExpr, ";").WithLocation(7, 27),
+                // (7,25): error CS8181: 'new' cannot be used with tuple type. Use a tuple literal expression instead.
+                //             var e = new ();
+                Diagnostic(ErrorCode.ERR_NewWithTupleTypeSyntax, "()").WithLocation(7, 25),
+                // (7,25): error CS8179: Predefined type 'System.ValueTuple`2' is not defined or imported
+                //             var e = new ();
+                Diagnostic(ErrorCode.ERR_PredefinedValueTupleTypeNotFound, "()").WithArguments("System.ValueTuple`2").WithLocation(7, 25));
```

Comment:
This error seems unnecessary. And it's confusing since it references `ValueTuple<T1, T2>`, presumably because we generated a 2-tuple with missing tokens.